javaslang/src/test/java/javaslang/collection/AbstractSeqTest.java

```diff
@@ -338,87 +336,119 @@ public void shouldGetLastElement() {
     @Test
     public void shouldNotFindIndexOfElementWhenSeqIsEmpty() {
         assertThat(empty().indexOf(1)).isEqualTo(-1);
+
+        assertThat(empty().indexOfOption(1)).isEqualTo(Option.none());
```

Comment:
Let's test both to be robust regarding refactoring. The implementation detail may change.